.github/workflows/build_pr_documentation.yml

```diff
@@ -9,8 +9,10 @@ concurrency:
 
 jobs:
   build:
-    uses: huggingface/doc-builder/.github/workflows/build_pr_documentation.yml@main
+    uses: huggingface/doc-builder/.github/workflows/build_pr_documentation.yml@use_gh_graphql
     with:
       commit_sha: ${{ github.event.pull_request.head.sha }}
       pr_number: ${{ github.event.number }}
       package: accelerate
+    secrets:
+      token: ${{ secrets.HUGGINGFACE_PUSH }}
```

Comment:
This is not an acceptable change, as secrets are only available on pull requests done by HF team member using a branch on the repo. They won't be available on any fork.